src/webgpu/api/validation/encoding/cmds/setBindGroup.spec.ts

```diff
@@ -125,6 +125,18 @@ g.test('state_and_binding_index')
     }
   });
 
+g.test('bind_group,device_mismatch')
+  .desc(
+    `
+    Tests setBindGroup cannot be called with a bind group created from another device
+    - x= setBindGroup {with, without} Uint32Array
+    `
+  )
+  .paramsSubcasesOnly(u =>
+    u.combine('encoderType', kProgrammableEncoderTypes).combine('mismatched', [true, false])
```

Comment:
nit: missing a param to pick which overload?

Reply:
Plan to test both sequence overload and Uint32Array overload in a same fn(). I can move it to params.

Reply:
Either way is okay, but params will probably make the code shorter because otherwise you'll need two entire command buffers